app/electron.js

```diff
@@ -59,6 +62,23 @@ function openAboutWindow() {
   }
 }
 
+function openHelpWindow() {
```

Comment:
also this is pretty much the same as about... 

I wonder if we can't unify them with some smart config?